vignettes/normals.Rmd

```diff
@@ -21,6 +21,7 @@ Climate Normals and Averages describe the average climate conditions specific to
 First we'll load the `weathercan` package for downloading the data and the `tidyr` package for unnesting the data (see below).
 
 ```{r setup, message = FALSE}
+options(rmarkdown.html_vignette.check_title = FALSE)
```

Comment:
Brilliant! Thanks! I've always wondered where to put this option but never took the time figure it out